modules/storage-expanding-flexvolume.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-flexvolume_{context}"]
+= Expanding FlexVolume with a supported driver
+
+When using FlexVolume to connect to your backend storage system, you can expand persistent storage volumes after they have already been created. This is done by manually updating the persistent volume claim (PVC) in {product-title}.
+
+FlexVolume allows expansion if the driver is set with `RequiresFSResize` to `true`. The FlexVolume can be expanded on Pod restart.
+
+Similar to other volume types, FlexVolume volumes can also be expanded when in use by a Pod.
+
+.Prerequisites
+
+* The underlying volume driver supports resize.
+* The driver is set with the `RequiresFSResize` capability to `true`.
+* Dynamic provisioning is used.
+* The controlling StorageClass has `allowVolumeExpansion` set to `true`.
+* The `PersistentVolumeClaimResize` admission plugin is enabled (optional).
```

Comment:
This is enabled by default in Openshift and hence I would drop the line about admission plugin from documentation.